meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I'd avoid inlining an import like this: `const Json2csvParser = require('json2csv').Parser`, put it at the top so it's clear what dependencies a file has. It also matters if you were using something like webpack to compile your assets